src/integrationTest/java/uk/gov/hmcts/reform/lrdapi/RetrieveCourtVenueDetailsIntegrationTest.java

```diff
@@ -137,7 +137,7 @@ void retrieveCourtVenues_WithCourtTypeIdGiven_ShouldReturnValidResponseAndStatus
             lrdApiClient.retrieveCourtVenueResponseForGivenRequest("?court_type_id=" + id,
                                                                    LrdCourtVenueResponse[].class, path);
 
-        assertThat(response).isNotEmpty().hasSize(9);
+        assertThat(response).isNotEmpty().hasSize(10);
```

Comment:
add assertion for EXTERNAL_SHORT_NAME value

Reply:
done